Templates/DefaultComponent/Template/Source/${Name}Component.cpp

```diff
@@ -0,0 +1,72 @@
+// {BEGIN_LICENSE}
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+// {END_LICENSE}
+
+#include <${SanitizedCppName}Component.h>
+
+#include <AzCore/Serialization/SerializeContext.h>
+#include <AzCore/Serialization/EditContext.h>
+#include <AzCore/RTTI/BehaviorContext.h>
+
+namespace ${SanitizedCppName}
+{
+    void ${SanitizedCppName}Component::Activate()
+    {
+        ${SanitizedCppName}RequestBus::Handler::BusConnect(GetEntityId());
+    }
+
+    void ${SanitizedCppName}Component::Deactivate()
+    {
+        ${SanitizedCppName}RequestBus::Handler::BusDisconnect(GetEntityId());
+    }
+
+    void ${SanitizedCppName}Component::Reflect(AZ::ReflectContext* context)
+    {
+        if (AZ::SerializeContext* serializeContext = azrtti_cast<AZ::SerializeContext*>(context))
+        {
+            serializeContext->Class<${SanitizedCppName}Component, AZ::Component>()
+                ->Version(1)
+                ;
+
+            if (AZ::EditContext* editContext = serializeContext->GetEditContext())
+            {
+                editContext->Class<${SanitizedCppName}Component>("${SanitizedCppName}Component", "[Description of functionality provided by this component]")
+                    ->ClassElement(AZ::Edit::ClassElements::EditorData, "")
+                    ->Attribute(AZ::Edit::Attributes::Category, "ComponentCategory")
+                    ->Attribute(AZ::Edit::Attributes::Icon, "Icons/Components/ComponentPlaceholder.svg")
```

Comment:
This needs to be updated to `Icons/Components/Component_Placeholder.svg` otherwise it won't be able to find it

Reply:
@cgalvan Why must the one with the underscore `_` be used here? I see both in that directory, actually. Also asking because in my tests, the one without the underscore seemed to work just fine as a component icon in the Inspector.

Reply:
Ah, I see now. The file without the `_` is identical, but unused. I will remove it since it doesn't appear anywhere in code. Thanks!